src/crimson/os/seastore/btree/fixed_kv_btree.h

```diff
@@ -511,6 +511,9 @@ class FixedKVBtree {
           &child_node);
       } else {
         if (i->get_val().pladdr.is_laddr()) {
+          assert((!std::is_base_of_v<typename internal_node_t::base_t, child_node_t>));
```

Comment:
Should this be moved to the above line in order to match `if constexpr (std::is_base_of_v<typename internal_node_t::base_t, child_node_t>)` above?